README.md

```diff
@@ -20,3 +20,7 @@ pnpm run dev
 ## Working on the theme
 
 If changes need to made for the theme, check out the [instructions for developing the theme alongside the docs](https://github.com/vuejs/vue-theme#developing-with-real-content).
+
+## Node.js Version Requirement
+
+This project uses JavaScript's new features like optional chaining, so it is recommended to use `v14.0.0` or higher.
```

Comment:
I'm not sure we need a separate section for this. Personally I think it would be sufficient just to mention the required version at the end of the `Contributing` section, just after the `pnpm` commands.

I also wonder whether we should include an `engines` section in `package.json`, to enforce the version explicitly rather than relying on documenting it?

Reply:
I generally approve your suggestion, It is briefer that just appending the the required version at the end of the `Contributing` section. However, since I never used the `engines` field before, I do not know whether it will print out warning or error messages or not, when an user hits `pnpm i`. According to the *npm* document:

> Unless the user has set the engine-strict config flag, this field is advisory only and will **only produce warnings when your package is installed as a dependency**.

[https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines)

Since this project is a standalone website instead of a third-party library, it will never be a dependency of other projects, so I am afraid that warnning will never be generated.

If the `engines` field is just a statement that will have any effects, I think it would be better to write down the requirement on the document. IMO, people like me is such lazy that probably won't take a look at the `packages.json` file after cloning the repository, and maybe have never thought that the the required version has been written in the `engines` field somewhere in the `packages.json` file. What I would do is to follow the installation guide and hope I can kick off the project ASAP, which will end up with a runtime error. If the required version have been written in the document, I will be informed and will get my Node.js updated first, since I have to visit the repository's home page before cloning.